packages/drivers/odsp-driver/src/odspDocumentService.ts

```diff
@@ -145,6 +148,10 @@ export class OdspDocumentService implements IDocumentService {
         if (this.odspResolvedUrl.summarizer) {
             this.hostPolicy = { ...this.hostPolicy, summarizerClient: true };
         }
+
+        if (this.hostPolicy.sessionOptions?.isolateSocketCache === true) {
```

Comment:
FYI - this will result in not sharing socket between summarizer & main client.

Reply:
Yep.  Fixed by generating the uuid per factory